.github/workflows/build_flavor.yml

```diff
@@ -64,7 +64,7 @@ jobs:
           fail-on-cache-miss: true
       - name: Write secureboot db arn for kms backed certificates
         if: ${{ inputs.use_kms }}
-        run: echo "${{ secrets.secureboot_db_kms_arn }}" > cert/gardenlinux-secureboot.db.arn
+        run: echo "${{ secrets.secureboot_db_kms_arn }}" > cert/gardenlinux-nightly-secureboot.db.arn
```

Comment:
This step is at the wrong place I think. It should be part of `.github/workflows/build_requirements.yml`. As this is not the case as of today we might:
- change `build_requirements.yml` to accept the required secret as optional.
- at least rename the file created here to `secureboot.db.arn`. In `build_requirements.yml` all files named `gardenlinux-nightly-*` are already linked and cached for later use e.g. in this step.